ref/Microsoft.Build.Tasks.Core/net/Microsoft.Build.Tasks.Core.cs

```diff
@@ -333,7 +333,6 @@ public partial class Exec : Microsoft.Build.Tasks.ToolTaskExtension
         [Microsoft.Build.Framework.OutputAttribute]
         public string StdOutEncoding { get { throw null; } set { } }
         protected override string ToolName { get { throw null; } }
-        public string UseUtf8Encoding { get { throw null; } set { } }
```

Comment:
Weird that the API addition in ToolTask does not show up.

Reply:
It's in M.B.Utilities.Core, so it's the next file. I was wondering if I should count this as a breaking change because someone could use UseUtf8Encoding for Exec without loading Utilities.Core, but I don't think there's anything that calls into it, so I think it's ok.